bin/alluxio

```diff
@@ -271,11 +271,13 @@ function main {
   "fs")
     CLASS="alluxio.cli.fs.FileSystemShell"
     CLASSPATH=${ALLUXIO_CLIENT_CLASSPATH}
+    ALLUXIO_SHELL_JAVA_OPTS+=" -Dalluxio.user.rpc.retry.max.duration=5s"
     runJavaClass "$@"
   ;;
   "fsadmin")
     CLASS="alluxio.cli.fsadmin.FileSystemAdminShell"
     CLASSPATH=${ALLUXIO_CLIENT_CLASSPATH}
+    ALLUXIO_SHELL_JAVA_OPTS+=" -Dalluxio.user.rpc.retry.max.duration=5s"
```

Comment:
usually users can configure their shell by setting alluxio-site.properties, but because of the order of precedence, this particular value won't be overridable from alluxio-site.properties, and it's source will show up as "SYSTEM_PROPERTY", which could be confusing to the user since the 5 seconds here is more like a default.

Reply:
make sense. I changed to set the `default` on CLIs. Let me know if that works